go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/Masterminds/semver v1.5.0
 	github.com/Masterminds/semver/v3 v3.1.0
 	github.com/ghodss/yaml v1.0.0
-	github.com/giantswarm/apiextensions/v3 v3.7.0
+	github.com/giantswarm/apiextensions/v3 v3.7.1-0.20201112171102-6ee24da912a5
```

Comment:
Can you check? https://github.com/giantswarm/apiextensions/pull/619

It will release 3.8.0 and I need it to merge some of my PRs too.

Reply:
Sure, I approved it 🚀